polyply/src/nonbond_engine.py

```diff
@@ -40,12 +39,20 @@ def _lennard_jones_force(dist, point, ref, params):
         the force vector
     """
     sig, eps = params
-    # doing this here with numba is faster than separate
+    # computing the vector here with numba is faster than separate
+    # the distance however again is faster computed with the cKDtree
+    # directly
```

Comment:
Parse-error

Reply:
cKD tree computes distances within a cut-off, right. So to get the LH force you need the distance but also the vector in the direction of the distance. You don't get the vector from the cKDtree. So you need to compute that extra. So you could write LJ_force(params, dist, vector) but computing the vector in LJ_force due to numba is faster. Alternatively you could also compute the distance from norm with numba here. However, again the cKD tree is even faster. Probably because it is hard-core optimized.

Reply:
Improve the comment, becuase it's barely legible at the moment.